packages/system/linstor/templates/satellites.yaml

```diff
@@ -46,3 +53,6 @@ spec:
               hostPath:
                 path: /var/etc/lvm/archive
                 type: DirectoryOrCreate
+            - name: host-run
+              hostPath:
+                path: /run
```

Comment:
no type on host-path catalogue. Which one will be chosen?

Reply:
Kubernetes uses DirectoryOrCreate by default. Just specifyed it expecitly

Reply:
I am asking to use the uniform approach on all `hostPath` entries. Thanks in advance